core/trino-main/src/main/java/io/trino/metadata/Metadata.java

```diff
@@ -858,4 +858,6 @@ default boolean isMaterializedView(Session session, QualifiedObjectName viewName
      * Returns writer scaling options for the specified table.
      */
     WriterScalingOptions getInsertWriterScalingOptions(Session session, TableHandle tableHandle);
+
+    void setEntityAuthorization(Session session, String ownedKind, List<String> name, TrinoPrincipal principal);
```

Comment:
Remove all other `set*Authorization` methods from this interface and implementations.

Reply:
Removed.